openedx/core/djangoapps/content/course_structures/tests.py

```diff
@@ -1,243 +0,0 @@
-
-        # Reload the data to ensure the init signal is fired to decompress the data.
-        cs = CourseStructure.objects.get(course_id=self.course.id)
-        self.assertEqual(cs.structure_json, structure_json)
-
-    def test_structure(self):
-        """
-        CourseStructure.structure should return the uncompressed, JSON-parsed course structure.
-        """
-        structure = {
-            'root': 'a/b/c',
-            'blocks': {
-                'a/b/c': {
-                    'id': 'a/b/c'
-                }
-            }
-        }
-        structure_json = json.dumps(structure)
-        cs = CourseStructure.objects.create(course_id=self.course.id, structure_json=structure_json)
-        self.assertDictEqual(cs.structure, structure)
-
-    def test_ordered_blocks(self):
-        structure = {
-            'root': 'a/b/c',
-            'blocks': {
-                'a/b/c': {
-                    'id': 'a/b/c',
-                    'children': [
-                        'g/h/i'
-                    ]
-                },
-                'd/e/f': {
-                    'id': 'd/e/f',
-                    'children': []
-                },
-                'g/h/i': {
-                    'id': 'h/j/k',
-                    'children': [
-                        'j/k/l',
-                        'd/e/f'
-                    ]
-                },
-                'j/k/l': {
-                    'id': 'j/k/l',
-                    'children': []
-                }
-            }
-        }
-        in_order_blocks = ['a/b/c', 'g/h/i', 'j/k/l', 'd/e/f']
-        structure_json = json.dumps(structure)
-        retrieved_course_structure = CourseStructure.objects.create(
-            course_id=self.course.id, structure_json=structure_json
-        )
-
-        self.assertEqual(retrieved_course_structure.ordered_blocks.keys(), in_order_blocks)
-
-    def test_block_with_missing_fields(self):
-        """
-        The generator should continue to operate on blocks/XModule that do not have graded or format fields.
-        """
-        # TODO In the future, test logging using testfixtures.LogCapture
-        # (https://pythonhosted.org/testfixtures/logging.html). Talk to TestEng before adding that library.
-        category = 'peergrading'
-        display_name = 'Testing Module'
-        module = ItemFactory.create(parent=self.section, category=category, display_name=display_name)
-        structure = _generate_course_structure(self.course.id)
-
-        usage_key = unicode(module.location)
-        actual = structure['structure']['blocks'][usage_key]
-        expected = {
-            "usage_key": usage_key,
-            "block_type": category,
-            "display_name": display_name,
-            "graded": False,
-            "format": None,
-            "children": []
-        }
-        self.assertEqual(actual, expected)
-
-    def test_generate_discussion_id_map(self):
```

Comment:
Just checking that someone validated that all of the discussion related tests below exist for the new version of this code.

Reply:
Those tests were created as part of previous PRs that introduced the new model and the new task for updating that model.  A search of `DiscussionsIdMapping` shows where it appears in test code.